### PR TITLE
Replace slackin with email link to the secretary

### PR DESCRIPTION
**Pull request by parndt**
We no longer use slackin so it makes sense to still give people a way to
ask to be added to our Slack.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -22,12 +22,12 @@ <h1>&#128075; Welcome!</h1>
 <section class="pure-g" id="join-the-community">
   <div class="pure-u-1">
     <h2>Join the Community</h2>
-    <p>For code chat, job listings, ruby help, keyboard nerdery, and banter, join us on <a href="http://slack.ruby.nz">Slack</a>. We're also on Twitter at <a href="http://twitter.com/rubynewzealand">@RubyNewZealand</a>.</p>
+    <p>For code chat, job listings, ruby help, keyboard nerdery, and banter, join us on [Slack](mailto:secretary@ruby.nz?subject=Can+I+have+RubyNZ+Slack+access+please). We're also on Twitter at <a href="http://twitter.com/rubynewzealand">@RubyNewZealand</a>.</p>
```
Comment (parndt) [inline index.html]:
```suggestion
    <p>For code chat, job listings, ruby help, keyboard nerdery, and banter, join us on <a href="mailto:secretary@ruby.nz?subject=Can+I+have+RubyNZ+Slack+access+please">Slack</a>. We're also on Twitter at <a href="http://twitter.com/rubynewzealand">@RubyNewZealand</a>.</p>
```

File: index.html [hunk 2]
```diff
@@ -22,12 +22,12 @@ <h1>&#128075; Welcome!</h1>
 <section class="pure-g" id="join-the-community">
   <div class="pure-u-1">
     <h2>Join the Community</h2>
-    <p>For code chat, job listings, ruby help, keyboard nerdery, and banter, join us on <a href="http://slack.ruby.nz">Slack</a>. We're also on Twitter at <a href="http://twitter.com/rubynewzealand">@RubyNewZealand</a>.</p>
+    <p>For code chat, job listings, ruby help, keyboard nerdery, and banter, join us on <a href="mailto:secretary@ruby.nz?subject=Can+I+have+RubyNZ+Slack+access+please">Slack</a>. We're also on Twitter at <a href="http://twitter.com/rubynewzealand">@RubyNewZealand</a>.</p>
```
Comment (parndt) [inline index.html]:
```suggestion
    <p>For code chat, job listings, ruby help, keyboard nerdery, and banter, join us on <a href="mailto:secretary@ruby.nz?subject='Can I have RubyNZ Slack access please?'">Slack</a>. We're also on Twitter at <a href="http://twitter.com/rubynewzealand">@RubyNewZealand</a>.</p>
```

File: index.html [hunk 3]
```diff
@@ -22,12 +22,12 @@ <h1>&#128075; Welcome!</h1>
 <section class="pure-g" id="join-the-community">
   <div class="pure-u-1">
     <h2>Join the Community</h2>
-    <p>For code chat, job listings, ruby help, keyboard nerdery, and banter, join us on <a href="http://slack.ruby.nz">Slack</a>. We're also on Twitter at <a href="http://twitter.com/rubynewzealand">@RubyNewZealand</a>.</p>
+    <p>For code chat, job listings, ruby help, keyboard nerdery, and banter, join us on <a href="mailto:secretary@ruby.nz?subject='Can I have RubyNZ Slack access please?'">Slack</a>. We're also on Twitter at <a href="http://twitter.com/rubynewzealand">@RubyNewZealand</a>.</p>
```
Comment (parndt) [inline index.html]:
```suggestion
    <p>For code chat, job listings, ruby help, keyboard nerdery, and banter, join us on <a href="mailto:secretary@ruby.nz?subject=Can I have RubyNZ Slack access please?">Slack</a>. We're also on Twitter at <a href="http://twitter.com/rubynewzealand">@RubyNewZealand</a>.</p>
```

**Review by dmitryrck — APPROVED** [review]
👏👏👏 Thanks!

**Review by warp — APPROVED** [review]

**Review by layerssss — APPROVED** [review]
Oh goodie. I'll setup an email forward from secretary@ruby.nz into my personal email so I don't miss those emails.

**Comment by parndt** [comment]
In future it might be good to have a web form for this instead of an email link but this is a practical first step at least.

**Pull request merged**